docs/_Research-Data-Management/03-md.md

```diff
@@ -57,46 +56,61 @@ In microbiology, metadata provides crucial contextual information about biologic
 
-Alternatively, we can hop over to the [MetadataStandards/Plant-associated microbiome biological-environmental metadata](https://github.com/NFDI4Microbiota/MetadataStandards/blob/main/Biological_Environmental/PlantAssoc_BioEnv_Metadata.md) where we can find a similar (but stripped down) checklist with some filed out examples for biological and environmental metadata. For the technical metadata corresponding to this example samling campaign we would refer the reader the [Amplicon sequencing](https://github.com/NFDI4Microbiota/MetadataStandards/blob/main/Technical/Amplicon_Technical_Metadata.md) and [Metagenome sequencing](https://github.com/NFDI4Microbiota/MetadataStandards/blob/main/Technical/Metagenome_Technical_Metadata.md) section of our [Technical MetadataStandards](https://github.com/NFDI4Microbiota/MetadataStandards/tree/main/Technical) part of repository.
+Alternatively, we can hop over to the [MetadataStandards/Plant-associated microbiome biological-environmental metadata](https://github.com/NFDI4Microbiota/MetadataStandards/blob/main/Biological_Environmental/PlantAssoc_BioEnv_Metadata.md) to find a similar (but stripped down) checklist with some pre-filled examples for biological and environmental metadata. For the technical metadata corresponding to this example sampling campaign we would refer the reader the [Amplicon sequencing](https://github.com/NFDI4Microbiota/MetadataStandards/blob/main/Technical/Amplicon_Technical_Metadata.md) and [Metagenome sequencing](https://github.com/NFDI4Microbiota/MetadataStandards/blob/main/Technical/Metagenome_Technical_Metadata.md) section of the [Technical MetadataStandards](https://github.com/NFDI4Microbiota/MetadataStandards/tree/main/Technical) part of the repository.
 
 By now, we should have a rough estimation of what kind of biological/environmental metadata we can collect before sampling, during sampling, and what could be collected during the processing of samples. 
 
+>[!IMPORTANT]
+>The metadata tables in the [NFDI4Microbiota Metadata Standards repository](https://github.com/NFDI4Microbiota/MetadataStandards) consist of suggestions of mandatory fields from different repository sources **as well as** additional metadata fields that will aid in adding more context-relevant information for your research data.
+>For the additional fields recommendations from metadata standards and individual suggestions have been discussed by NFDI4Microbiota consortium members and their affiliated collaboration partners.
+>If you see a field that should be considered a strong recommendation for your field of research, let us know by opening an issue. 
```

Comment:
I like the idea, we could add the URL to the new issue page creation https://github.com/NFDI4Microbiota/MetadataStandards/issues/new but if readers do not have a github account, there are directed to a login page which is weird, so maybe to the issues page? https://github.com/NFDI4Microbiota/MetadataStandards/issues ?

I do not expect a spamload of issues so I do not think that a dedicated label or template is necessary (what are your thoughts on that @magelm ?). Alternatively, there is a way to provide a bit of guidelines/pre-filled issue directly from the URL we give readers. This is what I did in https://cpauvert.shinyapps.io/validamiiid/ when you're using the share description button (see code for reference https://github.com/FAIR-MI/validamiiid/blob/c7f9031e06b442de263fce2258cedd1b66fb5d68/R/fct_conversion.R#L41)


```suggestion
>If you see a field that should be considered a strong recommendation for your field of research, let us know by opening an [issue](https://github.com/NFDI4Microbiota/MetadataStandards/issues). 
```

Reply:
@cpauvert : I think for now the normal issue reference is okay.
If we see issues that we can't relate to the appropriate table or technique, then we can think about a template for suggestions. 
The valiidamiiid template would need to be adapted for every metadatastandards table then, right? How much time/ effort is this roughly?